USAGE.md

```diff
@@ -45,9 +45,9 @@ See the [Hue Emulation](https://www.openhab.org/addons/integrations/hueemulation
     * The interactive REST API interface available through the openHAB 2 dashboard
     * PaperUI does not yet allow manipulation of tags
 
-#### Item Label Recommendation
+#### Item Label Recommendation and Requirement
 
-Matching of voice commands to Items happens based on the Item label (e.g. "Kitchen Light").
+Matching of voice commands to Items happens based on the Item label (e.g. "Kitchen Light").  (Note: Although, it is valid syntax to not have an item label, the string in quotes following the name.  For this skill to function properly a label is required.)
```

Comment:
I am ok with any change that simply sites a label is required.  I seem to not know my way around git very well and lost some of my changes.  

I deleted the one that had a non-working example as you requested.  I kept the other which I think just says a label isneeded.  I just can see the diffs properly to know if it is in the correct place.